### PR TITLE
BUGFIX: Fix Doctrine 1.3.3 compatibility

### LINKED ISSUE #3: TwoFactorAuthenticationService throws an exception for Doctrine\Persistence\ObjectManager
After updating Neos an error has occured. With doctrine/persistence 1.2.0 everything ist working well. But after updating there seems to be a problem with doctrine/persistence 1.3.3. Is there a fix coming?

**Exception**

`Exception #1265213849 in line 359 of /data/Packages/Framework/Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php: The object "Doctrine\Persistence\ObjectManager" which was specified as a property in the object configuration of object "Yeebase\TwoFactorAuthentication\Service\TwoFactorAuthenticationService" (automatically registered class) does not exist. Check for spelling mistakes and if that dependency is correctly configured.`

### PR DESCRIPTION
**Pull request by bwaidelich**
Fixes: #3

### CODE REVIEW

File: Classes/Service/TwoFactorAuthenticationService.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 use Doctrine\DBAL\Connection;
```
Comment (albe) [inline Classes/Service/TwoFactorAuthenticationService.php:5]: The one line above can be removed now
```use Doctrine\Common\Persistence\ObjectManager as DoctrineObjectManager;```

Comment (bwaidelich) [reply Classes/Service/TwoFactorAuthenticationService.php:5]: @albe Uh, which one? There is no `... as DoctrineObjectManager` line left?!

Comment (albe) [reply Classes/Service/TwoFactorAuthenticationService.php:5]: Need to view whole file or expand to top. It is there ;)

Comment (bwaidelich) [reply Classes/Service/TwoFactorAuthenticationService.php:5]: Doh, right. Done!

**Review by brgmn — APPROVED** [review]
👍

**Pull request merged**